microservices/frame_engine/cameraCV.py

```diff
@@ -1,54 +1,57 @@
 import cv2 as cv
 import sys
 import requests
+import time
 from datetime import datetime
 from pathlib import Path
-from config import *
+from requests import ReadTimeout, ConnectTimeout, HTTPError, Timeout, ConnectionError
+from config import W, H, FPS, urlapi, CORAL_DATA_DIR
 
 cap = cv.VideoCapture(0)
 
 cap.set(cv.CAP_PROP_FPS, FPS) # set Frame por Seconds
 cap.set(cv.CAP_PROP_FRAME_WIDTH,W) # set Width
 cap.set(cv.CAP_PROP_FRAME_HEIGHT,H) # set Height
 
-#Underwater Api url
+def sendApi(frame, framename):
```

Comment:
We should keep consistency on function names. The common pythonic way is with underscores, which is also what you've used elsewhere in the repo I think :) I know this will be automatically detected with a linter in the future, but just highlighting this one for now